tests/hpc/slurm_db.pm

```diff
@@ -25,10 +25,17 @@ sub run ($self) {
 
     $self->prepare_user_and_group();
 
-    # Install slurm
+    # If one wants to test unversioned slurm, one should not
+    # install slurm-node at all. Also slurm rpm does not
+    # provide mariadb as a dependency
+    if ($slurm_pkg eq 'slurm' and is_sle('=12-sp5')) {
+        zypper_call("in $slurm_pkg mariadb");
+    } else {
+        zypper_call("in $slurm_pkg $slurm_pkg-node");
```

Comment:
slurm|_slave has `zypper_call("in $slurm_pkg-node $slurm_pkg-munge");`

Reply:
slurm_slave has $slurm_pkg-munge by mistake. I'll remove that.